source/common/access_log/access_log_manager_impl.cc

```diff
@@ -41,8 +41,16 @@ AccessLogFileImpl::AccessLogFileImpl(Filesystem::FilePtr&& file, Event::Dispatch
   open();
 }
 
+Filesystem::FlagSet AccessLogFileImpl::defaultFlags() {
+  static constexpr Filesystem::FlagSet default_flags{
+      1 << Filesystem::File::Operation::Read | 1 << Filesystem::File::Operation::Write |
+      1 << Filesystem::File::Operation::Create | 1 << Filesystem::File::Operation::Append};
```

Comment:
why are the default flags for access logs to enable all these modes?

Reply:
The original logic has all modes flags.  And I don't want's to change this logic in this pr because it will take the potential risk for destroy some feature to `access_log` .

Reply:
I see; ok sounds good.